src/metabase/driver/firebolt.clj

```diff
@@ -247,20 +247,16 @@
   [_ [_ field min-val max-val]]
   [:nest [:between (sql.qp/->honeysql :firebolt field) (sql.qp/->honeysql :firebolt min-val) (sql.qp/->honeysql :firebolt max-val)]])
 
+; Choose post-filtered-active-tables method for firebolt, it's described as
+; Alternative implementation of `active-tables` best suited for DBs with little or no support for schemas
+(defmethod sql-jdbc.sync/active-tables :firebolt
+  [& args]
+  (apply sql-jdbc.sync/post-filtered-active-tables args))
 
-;; ------- Methods to handle Views, Describe database to not return Agg and Join indexes in Firebolt ----------------
-;; All the functions below belong to describe-table.clj which are all private in metabase and cant be called or
-;; extended directly. Hence, needed to implement the entire function chain
-;; As we should only return public tables that are not external, SHOW TABLES/SHOW VIEWS cannot be used.
-(defmethod driver/describe-database :firebolt
-  [_ {:keys [details] :as database}]
-  {:tables
-   (with-open [conn (jdbc/get-connection (sql-jdbc.conn/db->pooled-connection-spec database))]
-     (set/union
-      (set (for [{:keys [database table_name]} (jdbc/query {:connection conn} ["SELECT table_name from information_schema.tables WHERE table_schema LIKE 'public' AND table_type NOT LIKE 'EXTERNAL'"])]
-             {:name table_name :schema (when (seq database) database)}))
-      (set(for [{:keys [database table_name]} (jdbc/query {:connection conn} ["SELECT table_name from information_schema.views WHERE table_schema LIKE 'public'"])]
-            {:name table_name :schema (when (seq database) database)}))))})
+; Exclude information_schema schema from syncing
+(defmethod sql-jdbc.sync/excluded-schemas :firebolt
+  [_]
+  #{"information_schema"})
```

Comment:
This will just exclude them from the listing. They would still be able to access them via a custom query. As I see from other connectors, that's how it's usually done
https://github.com/metabase/metabase/blob/master/modules/drivers/snowflake/src/metabase/driver/snowflake.clj#L679
https://github.com/metabase/metabase/blob/master/src/metabase/driver/postgres.clj#L789
https://github.com/metabase/metabase/blob/master/src/metabase/driver/mysql.clj#L577

Reply:
I'd run this by the Product team - in our connectors for Tableau and Superset we have information schema exposed by default.

Reply:
I'll create a ticket for this. I'll enable infromation_schema for now

Reply:
https://packboard.atlassian.net/browse/FIR-37326